__tests__/app.test.js

```diff
@@ -69,16 +69,90 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-  test("200: reviews are sorted by newest first", () => {
+  test("200: reviews are sorted by newest first by default", () => {
     return request(app)
       .get("/api/reviews")
+      .expect(200)
       .then(({ body: { reviews } }) => {
-        expect(reviews).toBeSortedBy(reviews.created_at, {
+        expect(reviews).toBeSortedBy("created_at", {
           descending: true,
           coerce: true,
         });
       });
   });
+  describe("Queries", () => {
+    describe("Category", () => {
+      test("200: Filter by specified category", () => {
```

Comment:
This is really minor but I always get nervous when people use the word filtering in the test descriptions, its totally a me thing but I prefer something along the lines of can query by. I sometimes it leads to confusion and people tend to associate it with array filtering